Arguments: 
  /usr/local/bin/node /usr/local/bin/yarn add Var.sol

PATH: 
  /usr/local/bin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/homebrew/Caskroom/miniforge/base/bin:/opt/homebrew/Caskroom/miniforge/base/condabin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/foodchain/.cargo/bin

Yarn version: 
  1.22.19

Node version: 
  16.10.0

Platform: 
  darwin arm64

Trace: 
  Error: https://registry.yarnpkg.com/Var.sol: Not found
      at Request.params.callback [as _callback] (/usr/local/lib/node_modules/yarn/lib/cli.js:66145:18)
      at Request.self.callback (/usr/local/lib/node_modules/yarn/lib/cli.js:140890:22)
      at Request.emit (node:events:390:28)
      at Request.<anonymous> (/usr/local/lib/node_modules/yarn/lib/cli.js:141862:10)
      at Request.emit (node:events:390:28)
      at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/yarn/lib/cli.js:141784:12)
      at Object.onceWrapper (node:events:509:28)
      at IncomingMessage.emit (node:events:402:35)
      at endReadableNT (node:internal/streams/readable:1343:12)
      at processTicksAndRejections (node:internal/process/task_queues:83:21)

npm manifest: 
  No manifest

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile
